### PR TITLE
Fix/bugs 1,3

### PR DESCRIPTION
**Pull request by yk243**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -395,4 +395,6 @@ FodyWeavers.xsd
 *.msp
 
 # JetBrains Rider
-*.sln.iml
\ No newline at end of file
+*.sln.iml
+
+appsettings.json
```
Comment (ManeMag) [inline .gitignore]: appsettings must not be in gitignore. It's impossible to add new sections to them if needed

File: Squirrel/Controllers/AccountController.cs [hunk 2]
```diff
@@ -17,21 +17,22 @@ namespace Squirrel.Controllers
     [Route("api/[controller]/[action]")]
     public class AccountController : Controller
     {
-        private readonly UserManager<User> _userManager;
-        private readonly SignInManager<User> _signInManager;
-        private readonly IStringLocalizer<SharedResource> _localizer;
-        private readonly IEmailSender _emailSender;
+        private readonly UserManager<User>                  _userManager;
```
Comment (ManeMag) [inline Squirrel/Controllers/AccountController.cs:20]: Microsoft does not support such type of aligning

File: Squirrel/Program.cs [hunk 3]
```diff
@@ -76,6 +77,17 @@
     app.UseSwaggerUI();
     app.UseDeveloperExceptionPage();
 }
+app.UseCors(builder => builder
```
Comment (ManeMag) [inline Squirrel/Program.cs:80]: Can be simplified with AllowAnyOrigin, because I don't see any regularity between these urls

**Review by ManeMag — APPROVED** [review]
Approved with suggestions

File: Squirrel/Program.cs (hunk 3 above)
Comment (yk243) [reply Squirrel/Program.cs:80]: AllowAnyOrigin does not work with WithCredentials

**Pull request merged**